### PR TITLE
add roles based view function-frontend and backend

### PR DESCRIPTION
**Pull request by athakur73**
#62

### CODE REVIEW

**Comment by ashokpkumar** [comment]
Is should have caught this earlier. but please add the variable names without any space and in camelcase, this will help in setting the data in backend as well. like "rmgAdmin" instead of "RMG Admin"

File: frontend/frontend/src/app/add-employee/add-employee.component.html [hunk 1]
```diff
@@ -1,4 +1,4 @@
-<div class="add_employee_bg">
+<div class="add_employee_bg" *ngIf="roles=='RMG Admin'">
```
Comment (ashokpkumar) [inline frontend/frontend/src/app/add-employee/add-employee.component.html:1]: please change the variable names to camelcase without any space

File: frontend/frontend/src/app/add-projects/add-projects.component.html [hunk 2]
```diff
@@ -1,4 +1,4 @@
-<div class="add_project_bg">
+<div class="add_project_bg" *ngIf="roles=='Sales Team'">
```
Comment (ashokpkumar) [inline frontend/frontend/src/app/add-projects/add-projects.component.html:1]: please change values to camelcase "salesTeam" instead of "Sales Team"

**Review by ashokpkumar — CHANGES_REQUESTED** [review]

File: frontend/frontend/src/app/add-employee/add-employee.component.html (hunk 1 above)
Comment (athakur73) [reply frontend/frontend/src/app/add-employee/add-employee.component.html]: updated variable name

File: frontend/frontend/src/app/add-projects/add-projects.component.html (hunk 2 above)
Comment (athakur73) [reply frontend/frontend/src/app/add-projects/add-projects.component.html]: updated variable name in camelCase

**Pull request closed without merging**